tests/src/Interop/WinRT/NativeComponent/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake")
 include_directories("../Contracts")
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
+add_definition(-DWINRT_NO_MAKE_DETECTION)
```

Comment:
The original has add_compile_definitions here. Does that matter?

Reply:
This is an older style that works with the CMake versions that dotnet/coreclr uses.

Reply:
```suggestion
add_definitions(-DWINRT_NO_MAKE_DETECTION)
```